README.md

```diff
@@ -53,7 +53,7 @@ to COOL PCA accounts and resources.
 
 | Name | Version |
```

Comment:
I don't think the title of the README is accurate (I believe the repository was renamed). Do we want to fix that in this?

Reply:
🦅 👀   
Indeed we do! See https://github.com/cisagov/cool-pca-iam/pull/15/commits/ad53ca1db5591d4541b420f744b820917948ce14.

Reply:
Potent catch @mcdonnnj!